pyartifactory/models/repository.py

```diff
@@ -147,11 +180,13 @@ class LocalRepositoryResponse(LocalRepository):
 
 
 class VirtualRepository(BaseRepositoryModel):
+    """Models a virtual repository."""
+
     repositories: Optional[List[str]] = None
     artifactoryRequestsCanRetrieveRemoteArtifacts: bool = False
     debianTrivialLayout: bool = False
     keyPair: Optional[str] = None
-    pomRepositoryReferencesCleanupPolicy: PomRepoRefCleanupPolicy = PomRepoRefCleanupPolicy.discard_active_reference
+    pomRepositoryReferencesCleanupPolicy: PomRepoRefCleanupPolicy = PomRepoRefCleanupPolicy.discard_active_reference  # pylint: disable=line-too-long

```

Comment:
Instead of disabling the **line-too-long** error here. You can do it directly in the pylintrc. It would prevent you from addind this comment to each line, yet formatted by black, but too long.